.github/workflows/nightly.yml

```diff
@@ -0,0 +1,76 @@
+name: nightly
```

Comment:
This is per closed, PR, not nightly, right?  I'd expect to see a schedule if it was time driven.

Reply:
Ah yes, this is per PR. I think at this point in time that is more sensible as we have more nights as PRs atm.

I couldn't think of a better name. Also I think nightly is still kinda ok.

Reply:
Yeah, the name is fine, I just wanted to make sure there wasn't a difference in expected behavior versus what I was reading